apps-rendering/src/components/Layout/NewsletterSignUpLayout.tsx

```diff
@@ -0,0 +1,147 @@
+import type { ArticleFormat } from '@guardian/libs';
+import {
+	brand,
+	breakpoints,
+	from,
+	neutral,
+	remSpace,
+} from '@guardian/source-foundations';
+import { SvgGuardianLogo } from '@guardian/source-react-components';
+import Body from 'components/ArticleBody';
+import { tabletContentWidth } from 'components/editions/styles';
+import Footer from 'components/Footer';
+import Headline from 'components/Headline';
+import Logo from 'components/Logo';
+import MainMedia from 'components/MainMedia';
+import RelatedContent from 'components/RelatedContent';
+import Standfirst from 'components/Standfirst';
+import { getFormat } from 'item';
+import type { Standard as StandardItem } from 'item';
+import type { FC, ReactNode } from 'react';
+import {
+	darkModeCss,
+	onwardStyles,
+	sidePadding,
+	wideContentWidth,
+} from 'styles';
+
+// ----- Styles ----- //
+const backgroundStyles = (format: ArticleFormat): SerializedStyles => css`
+	background-color: ${background.articleContent(format)};
+
+	${darkModeCss`
+        background-color: ${background.articleContentDark(format)}
+    `}
+`;
+
+const sectionStyles: SerializedStyles = css`
+	margin: 0 auto;
+	position: relative;
+
+	${from.phablet} {
+		max-width: ${tabletContentWidth}px;
+	}
+
+	${from.desktop} {
+		max-width: ${wideContentWidth}px;
+	}
+
+	${sidePadding.styles}
+`;
+
+const borderStyles = css`
+	${from.wide} {
+		width: ${breakpoints.wide}px;
+		margin: 0 auto;
+	}
+`;
+
+const bannerStyle: SerializedStyles = css`
+	background-color: ${brand[400]};
+	padding: ${remSpace[2]} 0 ${remSpace[9]};
+`;
+
+const figureStyle: SerializedStyles = css`
+	display: block;
+	width: 11rem;
+	position: relative;
+`;
+
+const logoWrapper: SerializedStyles = css`
+	display: flex;
+	width: 65%;
+	margin-bottom: ${remSpace[1]};
+`;
+
+const badgeWrapper: SerializedStyles = css`
+	font-size: 1rem;
+	display: flex;
+	width: 100%;
+`;
+
+// ----- Component ----- //
+interface Props {
+	item: StandardItem;
+	children: ReactNode[];
+}
+
+const NewsletterSignUpLayout: FC<Props> = ({ item, children }) => {
+	const format = getFormat(item);
+
+	return (
+		<main css={backgroundStyles(format)}>
+			<article className="js-article" css={borderStyles}>
+				<header>
+					<div css={bannerStyle}>
+						<div css={sectionStyles}>
+							<figure
+								css={figureStyle}
+								role="figure"
+								aria-label="The Guardian Newsletters"
+							>
+								<span css={logoWrapper}>
+									<SvgGuardianLogo
+										textColor={neutral[100]}
+										width={200}
+									/>
+								</span>
+								<span css={badgeWrapper}>
+									<NewsletterBadge />
+								</span>
+							</figure>
+						</div>
+					</div>
+
+					<div css={sectionStyles}>
+						<Headline item={item} />
+						<Standfirst item={item} />
+						<Logo item={item} />
+					</div>
+				</header>
+
+				<section css={sectionStyles}>
+					<Body format={item}>{children}</Body>
```

Comment:
For now, the article bodies include the old-style article embeds as a fall back for the AR version of the SignUp pages.

If we follow the method used on DCR, the Newsletter Sign Up component will be be put on the page instead of having an ArticleBody. This would need to wait until the AR Newsletter sign up component was full functional.

The SignUp pages have the same tags used to place the in in-article sign-up blocks, so the MAPI data for them should include the `PromotedNewsletter` meta data that can be used to set the props for a Newsletter sign up component within the layout.